examples/ex1_standalone_app/build.gradle

```diff
@@ -14,5 +14,8 @@ android {
 }
 
 dependencies {
+    // External users should use the repo dependency below instead of the
+    // project dependency.
+    //compile 'com.google.android.mobly:mobly-snippet-lib:1.0.0'
```

Comment:
Something along the line of 'This dep is for building directly against the src here. If you're not building against the tree, use * instead'?